lib/compat/wordpress-6.8/preload.php

```diff
@@ -10,11 +10,23 @@
  */
 function gutenberg_block_editor_preload_paths_6_8( $paths, $context ) {
 	if ( 'core/edit-site' === $context->name ) {
-		if ( ! empty( $_GET['postId'] ) ) {
+		if ( ! empty( $_GET['postId'] ) && ! empty( $_GET['postType'] ) ) {
```

Comment:
The `$context` has a `post` property, only populated in the post editor. I think we should do the same for the `core/edit-site`, but when it makes sense. Currently, that would be a page post type.

Reply:
@Mamaduka yes, that's a good idea. We can't do it here correctly, but I did it in the backport: #7695